x-pack/plugin/src/yamlRestTest/resources/rest-api-spec/test/ml/sparse_vector_search.yml

```diff
@@ -89,6 +89,29 @@ setup:
         model_id: text_expansion_model
         wait_for: started
 
+---
+teardown:
+  - requires:
```

Comment:
Why do we have requires in the tear down? Isn't it enough to remove indices and ignore 404s?